src/chrome/utils/pastebin.ts

```diff
@@ -0,0 +1,59 @@
+import { getItemAsync } from "./storage";
+import { Storage } from "../../constants";
+
+export async function postPastebin(encryptQuery: string) {
```

Comment:
@fairhurt I ended making a new request function without any react stuff and it seems to be working good. Right click and extension decrypt/encrypt work via pastebin. Never used react before so I just tried plain js and it seems to work?